diff --git a/tests/gptl_avail.c b/tests/gptl_avail.c
index dfa17f7..e42f81a 100644
--- a/tests/gptl_avail.c
+++ b/tests/gptl_avail.c
@@ -38,11 +38,12 @@ int main ()
     ret = GPTLfinalize ();
   }
 
+  /*
   if (GPTLsetoption (GPTL_DCMRT, 1) == 0) {
     printf("%-20s %-10d %s\n", "GPTL_DCMRT", GPTL_DCMRT, "L1 Miss rate (fraction)");
     ret = GPTLinitialize ();
     ret = GPTLfinalize ();
-  }
+  }*/
 
   if (GPTLsetoption (GPTL_LSTPDCM, 1) == 0) {
     printf("%-20s %-10d %s\n", "GPTL_LSTPDCM", GPTL_LSTPDCM, "Load-store instructions per L1 miss");
