diff --git a/configure.ac b/configure.ac
index 3e0eb73..32956b9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -199,8 +199,8 @@ AS_IF([test x$enable_papi = xyes], [
     # If we have PAPI library, check /proc/sys/kernel/perf_event_paranoid
     # to see if we have permissions.
     if test -f /proc/sys/kernel/perf_event_paranoid; then
-      if test `cat /proc/sys/kernel/perf_event_paranoid` != 1; then
-         AC_MSG_ERROR([PAPI library found, but /proc/sys/kernel/perf_event_paranoid != 1
+      if test `cat /proc/sys/kernel/perf_event_paranoid` <= 1; then
+         AC_MSG_ERROR([PAPI library found, but /proc/sys/kernel/perf_event_paranoid <= 1
                             try sudo sh -c 'echo 1 >/proc/sys/kernel/perf_event_paranoid'])
       fi
     fi
