diff --git a/CMakeLists.txt b/CMakeLists.txt
index 13c5cf7f..d063cfef 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,6 +4,10 @@ project( eckit CXX )
 
 set( CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/../ecbuild/cmake")
 
+#Find ecbuild so that cmake can directly configure this package
+find_package(ecbuild REQUIRED)
+set(CMAKE_MODULE_PATH ${ECBUILD_CMAKE_DIR} ${CMAKE_MODULE_PATH})
+
 include( ecbuild_system NO_POLICY_SCOPE )
 
 ecbuild_requires_macro_version( 2.9 )
 
