diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7f1723d..4c2bc9f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -35,7 +35,8 @@ ecbuild_requires_macro_version( 1.6 )
 
 ###############################################################################
 # local project
-
+include( GNUInstallDirs )
+set(INSTALL_LIB_DIR "${CMAKE_INSTALL_LIBDIR}")
 ecbuild_declare_project()
 
 # =========================================================================================
