diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 94fcb94..28b6ac6 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1241,7 +1241,7 @@ endif ()
 #-----------------------------------------------------------------------------
 set (_PKG_CONFIG_PREFIX ${CMAKE_INSTALL_PREFIX})
 set (_PKG_CONFIG_EXEC_PREFIX \${prefix})
-set (_PKG_CONFIG_LIBDIR \${exec_prefix}/lib)
+set (_PKG_CONFIG_LIBDIR \${exec_prefix}/${CMAKE_INSTALL_LIBDIR})
 set (_PKG_CONFIG_INCLUDEDIR \${prefix}/include)
 set (_PKG_CONFIG_LIBNAME "${HDF5_LIB_CORENAME}")
 set (_PKG_CONFIG_VERSION "${HDF5_PACKAGE_VERSION}")
