diff --git a/fortran/src/CMakeLists.txt b/fortran/src/CMakeLists.txt
index 5caae5a..bbf8e07 100644
--- a/fortran/src/CMakeLists.txt
+++ b/fortran/src/CMakeLists.txt
@@ -291,7 +291,7 @@ if (NOT ONLY_SHARED_LIBS)
       PRIVATE
           "${HDF5_F90_SRC_SOURCE_DIR};${CMAKE_Fortran_MODULE_DIRECTORY}/static;${HDF5_F90_BINARY_DIR};$<$<BOOL:${HDF5_ENABLE_PARALLEL}>:${MPI_Fortran_INCLUDE_DIRS}>"
       INTERFACE
-          "$<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include/static>"
+          "$<INSTALL_INTERFACE:${HDF5_INSTALL_INCLUDE_DIR}/static>"
   )
   target_compile_definitions(${HDF5_F90_LIB_TARGET}
       PRIVATE
@@ -331,7 +331,7 @@ if (BUILD_SHARED_LIBS)
       PRIVATE
           "${HDF5_F90_SRC_SOURCE_DIR};${CMAKE_Fortran_MODULE_DIRECTORY}/shared;${HDF5_F90_BINARY_DIR};$<$<BOOL:${HDF5_ENABLE_PARALLEL}>:${MPI_Fortran_INCLUDE_DIRS}>"
       INTERFACE
-          "$<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include/shared>"
+          "$<INSTALL_INTERFACE:${HDF5_INSTALL_INCLUDE_DIR}/shared>"
   )
   target_compile_definitions(${HDF5_F90_LIBSH_TARGET}
       PUBLIC
