diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1eae19ea..8c9ddb91 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required( VERSION 3.6 FATAL_ERROR )
 
 project( eckit CXX )
-
+find_package(ecbuild REQUIRED)
 set( CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/../ecbuild/cmake")
 
 include( ecbuild_system NO_POLICY_SCOPE )
