diff --git a/setup.py b/setup.py
index f9c8d98..48a72f4 100644
--- a/setup.py
+++ b/setup.py
@@ -14,16 +14,16 @@ def read(path):
 #   https://packaging.python.org/guides/single-sourcing-package-version/
 def parse_version_from(path):
     version_file = read(path)
-    version_match = re.search(r"^__version__ = '(.*)'", version_file, re.M)
+    version_match = re.search(r"^\s*__version__\s*=\s*[\"'](.*)[\"']", version_file, re.M)
     if version_match is None or len(version_match.groups()) > 1:
         raise ValueError("couldn't parse version")
     return version_match.group(1)
 
 
 setuptools.setup(
-    name='odyssey',
+    name='pyodc',
     version=parse_version_from('codc.py'),
-    description='Python interface to odc for encoding/decoding ODB files'
+    description='Python interface to odc for encoding/decoding ODB files',
     long_description=read('README.rst') + read('CHANGELOG.rst'),
     author='European Centre for Medium-Range Weather Forecasts (ECMWF)',
     author_email='software.support@ecmwf.int',
